library/alloc/src/rc.rs

```diff
@@ -535,7 +540,7 @@ impl<T> Rc<T> {
     /// # Ok::<(), std::alloc::AllocError>(())
     /// ```
     #[unstable(feature = "allocator_api", issue = "32838")]
-    // #[unstable(feature = "new_uninit", issue = "63291")]
```

Comment:
Please keep this line. As long as multiple unstable features can't be documented, at least the comment should stay.

Reply:
Oh, sorry